app/grandchallenge/evaluation/models.py

```diff
@@ -616,6 +641,15 @@ class Meta:
             ("configure_algorithm_phase", "Configure Algorithm Phase"),
         )
 
+    def get_selectable_gpu_type_choices_evaluation(self):
+        choices = {GPUTypeChoices.NO_GPU, GPUTypeChoices.T4}
```

Comment:
If you did that in `settings.py` you would need to import from the application, which won't work (or you will need to refactor for circular dependencies). I think it's okay there for now, otherwise you can turn it into a module constant, e.g.

```python
DEFAULT_GPU_CHOICES = [GPUTypeChoices.NO_GPU, GPUTypeChoices.T4]
```

Changed to a list rather than a set here as ordering is often important.

Reply:
Looking at this further I think it makes most sense to define this as the default to the field.

Reply:
Settings are already imported here through `from django.conf import settings`, so circular dependency is not an issue.

> Changed to a list rather than a set here as ordering is often important.

I was looking into OrderedSets, but it requires another package. 

> Looking at this further I think it makes most sense to define this as the default to the field.  

I'll go with this for now!